Jfile

```diff
@@ -18,7 +18,7 @@ bundle 'pusher.js' do
   license 'pusher-licence.js'
 
   file 'build/umd_header.js'
-
+  file 'runtime.js'
```

Comment:
`runtime.js` should probably be included after `pusher.js`.

`pusher.js` defines the `Pusher` global. So if the contents of `runtime.js` were executed prior to the code in `pusher.js` then the `Pusher` global will not yet exist.